weaviate/collections/classes/aggregate.py

```diff
@@ -258,12 +259,14 @@ def __init__(self, property_: str) -> None:
 
     def text(
         self,
-        count: bool = False,
-        top_occurrences_count: bool = False,
-        top_occurrences_value: bool = False,
+        count: Optional[bool] = None,
+        top_occurrences_count: Optional[bool] = None,
+        top_occurrences_value: Optional[bool] = None,
```

Comment:
couldn't we thread everything False as "return all"? Why would you make a query when you disable every possible return?

Reply:
Yeh, I think you're right. I'll change to make it simpler